variable/operations.cpp

```diff
@@ -36,16 +36,14 @@ Variable copy(const Variable &var, Variable &&out) {
 }
 
 Variable masked_to_zero(const Variable &var, const Variable &mask) {
-  return transform(var, mask, element::convertMaskedToZero);
+  return variable::transform(var, mask, element::convertMaskedToZero,
+                             "masked_to_zero");
 }
 
 namespace geometry {
 Variable position(const Variable &x, const Variable &y, const Variable &z) {
-  return transform(x, y, z, element::geometry::position);
+  return transform(x, y, z, element::geometry::position, "linalg.position");
```

Comment:
Why 'linalg'?

Reply:
No idea, I think I wanted to rename `geometry` to `linalg` for consistency with numpy, but then didn't. I'll change.